apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -907,7 +923,7 @@ export function buildMachine(
                 PAPER_JAM: 'jammed',
               },
               after: {
-                [DELAY_BEFORE_DECLARING_REAR_JAM_MS]: 'jammed',
+                [delays.DELAY_BEFORE_DECLARING_REAR_JAM_MS]: 'jammed',
```

Comment:
These could probably also just reference the machine's configured delay with that key now:
```ts
after: {
  DELAY_BEFORE_DECLARING_REAR_JAM_MS: 'jammed'
}
```
Not really any functional difference, so take it or leave it

Reply:
Mostly copied this construct over for consistency with VxScan. I don't think we need the interface but it's nice to have all the delays more obviously located in a single place (whereas before they were in a random constants file).